preprocessing/preprocess_flex.js

```diff
@@ -26,8 +26,22 @@ function isSeparator(char) {
 // word - the data associated with a single word of the source text, 
 //   structured as in FLEx
 // return true if the word is tagged as punctuation within FLEx
+// If the word is marked as type "punct" but the word is alphanumeric,
```

Comment:
I'd like to figure out what's up with this preprocess_flex diff before you merge. I'll poke around and make sure BrownCLPS is fully merged into CofanALDP (it might not be). Worst case, we can hop on a call and resolve some merge conflicts together :)

Reply:
Just by looking at the files in the master branch of the Cofan repo right now, the preprocess_flex file seems up-to-date, so I'm not sure why making this PR didn't catch that..

Reply:
Hm... Ok, maybe it really is the best approach to merge this in and see what happens.